audit/nodes.go

```diff
@@ -52,9 +54,15 @@ func ProcessManual(ctx context.Context, data *logical.LogInput, ids []eventlogge
 
 	// Process nodes in order, updating the event with the result.
 	// This means we *should* do:
-	// 1. formatter (temporary)
-	// 2. sink
+	// 1. filter (optional if configured)
+	// 2. formatter (temporary)
```

Comment:
We've got a question about whether or not we should allow test messages (which is what would cause `ProcessManual` to be requested if a filter node is part of the pipeline. So this code may change in a future PR based on the outcomes of that.

To answer the question though 😄 if the filter node filters the message that ends the pipeline, so the formatter and sink nodes never get called.